tests/dummy/app/components/live-query.ts

```diff
@@ -0,0 +1,14 @@
+import Component from '@glimmer/component';
+import { use, useLiveQuery } from 'ember-orbit';
+
+interface Args {
+  id?: string;
+}
+
+export default class extends Component<Args> {
+  @use data = useLiveQuery((q) =>
+    this.args.id
+      ? q.findRecord({ type: 'planet', id: this.args.id })
```

Comment:
does this re-run if the args change?

Reply:
I think it will not. After some thoughts, I think we should wait for `invokeHelper`/`use` apis to settle.